map-style-change/index.html

```diff
@@ -44,26 +44,28 @@
       <label for="dark">Dark</label>
     </div>
     <script src="https://unpkg.com/maplibre-gl"></script>
-    <script src="https://unpkg.com/amazon-location-helpers"></script>
+    <script src="AUTH URL TBD"></script>
     <script>
       async function initializeMap() {
+        const identityPoolId = "<Identity Pool ID>";
+        const mapName = "map-style-change-example-1";
+        const region = identityPoolId.split(":")[0];
+
+        // Create an authentication helper instance using credentials from Cognito
+        const authHelper = await amazonLocationAuthHelper.withIdentityPoolId(identityPoolId);
```

Comment:
Should we use API key here?

Reply:
Not yet, since we don't have the ability to create API keys via CFN.